libs/error-stack/src/context.rs

```diff
@@ -56,6 +58,7 @@ use crate::Report;
 /// # assert!(err.contains::<io::Error>());
 /// # assert!(err.contains::<ConfigError>());
 /// ```
+#[deprecated(note = "Use `core::error::Error` instead", since = "0.6.0")]
```

Comment:
should we say instead `Use core::error::Error + Send + Sync + 'static` instead? Because that's the newly required trait bound.

Reply:
In most cases the user just implements `Context`. Only if you want to be generic over the error you need those bounds. I think this is good enough.